influunt-app/features/perfis/perfis.feature

```diff
@@ -27,8 +27,8 @@ Funcionalidade: Verificar o perfil e seus acessos
   Cenário: Com o perfil básico não posso ter acesso a certas funcionalidades
     Dado o usuário logue no sistema com usuário "mobilab" e perfil "basico"
     Então o usuário deve ser enviado para a tela de dashboard
-    Então o usuário não deve ter acesso ao menu "auditoria"
-    E o usuário não deverá ter acesso a listagem de "auditorias"
+    # Então o usuário não deve ter acesso ao menu "auditoria"
+    # E o usuário não deverá ter acesso a listagem de "auditorias"
```

Comment:
Acho que não irá ter mais menu auditoria pelo que entendi, caso for então, pode remover essas steeps.